docs/_utils/redirects.yaml

```
@@ -2,5 +2,8 @@
 #old path: new path
 #
 
+/stable/resources/scyllaclusters/nodeoperations/restore.html: https://operator.docs.scylladb.com/stable/resources/scyllaclusters/nodeoperations/restore.html
+/stable/manager.html: https://operator.docs.scylladb.com/stable/architecture/manager.html
```

I believe we could use relative links as targets.
```suggestion
/stable/resources/scyllaclusters/nodeoperations/restore.html: /stable/resources/scyllaclusters/nodeoperations/restore.html
/stable/manager.html: /stable/architecture/manager.html
```

---

We could use both, see https://github.com/scylladb/scylladb/blob/master/docs/_utils/redirects.yaml. But I agree with you that it looks cleaner with relative links as targets. Committed.